### PR TITLE
Update changesets for release

### PR DESCRIPTION
**Pull request by michaeltaranto**
Refining some of the changesets in preparation for release

### CODE REVIEW

File: .changeset/breezy-fireants-yawn.md [hunk 1]
```diff
@@ -4,6 +4,34 @@
...
-To support the pre-compilation of Braid, it is now a requirement that consumers are on React 17 or later. For more information on the differences between v16 and v17, [see the React blog].
+To support the pre-compilation of Braid, it is now a requirement that consumers are on React 17 or later.
+Originally the plan was to drop React 17 as well, however the team has been able to maintain support for a little longer — but there is a catch (see migration guide below).
 
-[see the React blog]: https://reactjs.org/blog/2020/10/20/react-v17.html
+**MIGRATION GUIDE:**
+### React 16 consumers
+As this version is no longer supported it is a requirement that consumers upgrade.
+For more information on the differences between v16 and v17, [see the React blog][react17].
+We strongly encourage teams, take this opportunity and to upgrade to [v18][react18] as well.
```
Comment (askoufis) [inline .changeset/breezy-fireants-yawn.md:14]:
```suggestion
We strongly encourage teams to take this opportunity to upgrade to [v18][react18] as well.
```

Comment (mrm007) [inline .changeset/breezy-fireants-yawn.md:14]:
```suggestion
We strongly encourage teams to take this opportunity and upgrade to [v18][react18] as well.
```

File: .changeset/new-cheetahs-build.md [hunk 2]
```diff
@@ -4,5 +4,5 @@
 
 Remove treat support
 
-[Treat](https://seek-oss.github.io/treat/) is deprecated and has been superseded by [vanilla-extract](https://vanilla-extract.style/).
-`.treat.ts` files can no longer be used to for styling and should be converted to `.css.ts` (vanilla-extract) files.
+[Treat](https://seek-oss.github.io/treat/) has been deprecated for a nearly 2 years, superseded by [vanilla-extract](https://vanilla-extract.style/).
```
Comment (askoufis) [inline .changeset/new-cheetahs-build.md:7]:
```suggestion
[Treat](https://seek-oss.github.io/treat/) has been deprecated for nearly 2 years, superseded by [vanilla-extract](https://vanilla-extract.style/).
```

File: .changeset/new-cheetahs-build.md [hunk 3]
```diff
@@ -4,5 +4,5 @@
...
+`.treat.ts` files can no longer be used to for styling and should be converted to `.css.ts` (vanilla-extract) stylesheets.
```
Comment (mrm007) [inline .changeset/new-cheetahs-build.md:8]:
```suggestion
`.treat.ts` files can no longer be used for styling and should be converted to `.css.ts` (vanilla-extract) stylesheets.
```

File: .changeset/breezy-fireants-yawn.md [hunk 4]
```diff
@@ -4,6 +4,34 @@
...
+This can be done by adding the following to the webpack config:
+
+```ts
+{
+  resolve: {
+    fallback: {
+      'react/jsx-runtime': require.resolve('react/jsx-runtime'),
+    },
+  },
+}
```
Comment (mrm007) [inline .changeset/breezy-fireants-yawn.md:27]:
```suggestion
// sku.config.ts
{
  dangerouslySetWebpackConfig: (config) => {
    config.resolve.fallback = {
      ...config.resolve.fallback,
      'react/jsx-runtime': require.resolve('react/jsx-runtime'),
    };

    return config;
  },
}
```

Might also be worth including a version with `webpack-merge`:

```ts
// sku.config.ts
{
  dangerouslySetWebpackConfig: (config) =>
    webpackMerge(config, {
      resolve: {
        fallback: {
          'react/jsx-runtime': require.resolve('react/jsx-runtime'),
        },
      },
    }),
}
```

**Review by mrm007 — APPROVED** [review]

**Review by askoufis — APPROVED** [review]

File: .changeset/breezy-fireants-yawn.md [hunk 5]
```diff
@@ -4,6 +4,44 @@
...
+    webpackMerge(config, {
+      resolve: {
+        fallback: {
+          'react/jsx-runtime': require.resolve('react/jsx-runtime'),
+        },
+      },
+    }),
+}
+```
+We recommend using [webpack-merge] to ensure the both configurations are deep merged correctly.
```
Comment (askoufis) [inline .changeset/breezy-fireants-yawn.md:35]:
```suggestion
We recommend using [webpack-merge] to ensure both configurations are deep merged correctly.
```

**Pull request merged**